### ISSUE TITLE
Using jupyter labextension manager to install jupyterlab-recents gives error, but conda install works? 

### ISSUE THREAD

**Issue by richlysakowski** [jupyterlab-contrib/jupyterlab-recents#6]
## Description

I wanted to install the package "jupyterlab-recents" the same way that I installed the package "jupyterlab-code-snippets"

The following command WORKED for "jupyterlab-code-snippets" using the jupyter labextension manager, but not for "jupyterlab-recents".

```
C:\Users\PowerUser\Desktop>jupyter labextension install jupyterlab-code-snippets
Building jupyterlab assets (production, minimized)
```

However, the following command failed with a jupyterlab-recents version error

```C:\Users\PowerUser\Desktop>jupyter labextension install jupyterlab-recents```

Then I tried the ```conda install -c conda-forge jupyterlab-recents``` and IT WORKED. 

HERE IS THE RESULT: 
C:\Users\PowerUser\Desktop>conda install -c conda-forge jupyterlab-recents
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: C:\ProgramData\Anaconda3\envs\topss

  added / updated specs:
    - jupyterlab-recents

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    jupyterlab-recents-3.0.1   |     pyhd8ed1ab_0          17 KB  conda-forge
    ------------------------------------------------------------
                                           Total:          17 KB

The following NEW packages will be INSTALLED:

  jupyterlab-recents conda-forge/noarch::jupyterlab-recents-3.0.1-pyhd8ed1ab_0 None

Proceed ([y]/n)? y

Downloading and Extracting Packages
jupyterlab-recents-3 | 17 KB     | ###################################################################################################################################### | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
Retrieving notices: ...working... done

##############################

I don't understand why the approach using "jupyter labextension manager" did not work. 

HERE IS THE OUTPUT using "jupyter labextension manager" 

## Reproduce

Type the command: 

```C:\Users\PowerUser\Desktop>jupyter labextension install jupyterlab-recents```

Here is the result: 

```
An error occurred.
ValueError: The extension "jupyterlab-recents" does not yet support the current version of JupyterLab.

Conflicting Dependencies:
JupyterLab              Extension      Package
>=3.4.7 <3.5.0          >=2.0.0 <3.0.0 @jupyterlab/application
>=5.4.7 <5.5.0          >=4.0.0 <5.0.0 @jupyterlab/coreutils
>=3.4.7 <3.5.0          >=2.0.0 <3.0.0 @jupyterlab/docmanager
>=3.4.7 <3.5.0          >=2.0.0 <3.0.0 @jupyterlab/mainmenu
>=6.4.7 <6.5.0          >=5.0.2 <6.0.0 @jupyterlab/services
>=3.4.7 <3.5.0          >=2.0.1 <3.0.0 @jupyterlab/statedb
See the log file for details:  C:\Users\POWERU~1\AppData\Local\Temp\jupyterlab-debug-8_tprghj.log
```
HERE IS THE LOG FILE: 

```Node v18.9.1

Yarn configuration loaded.
> C:\ProgramData\Anaconda3\envs\topss\npm.CMD pack jupyterlab-recents
npm notice 
npm notice package: jupyterlab-recents@2.0.0
npm notice === Tarball Contents === 
npm notice 1.6kB  LICENSE        
npm notice 512B   README.md      
npm notice 145B   lib/index.d.ts 
npm notice 10.0kB lib/index.js   
npm notice 1.5kB  package.json   
npm notice 0B     style/index.css
npm notice === Tarball Details === 
npm notice name:          jupyterlab-recents                      
npm notice version:       2.0.0                                   
npm notice filename:      jupyterlab-recents-2.0.0.tgz            
npm notice package size:  4.6 kB                                  
npm notice unpacked size: 13.7 kB                                 
npm notice shasum:        4cb4b8a11a9e33cf10583ac3a99bd1133b1933bb
npm notice integrity:     sha512-6NwEzLvQCD7VU[...]3anieIo7bt8uQ==
npm notice total files:   6                                       
npm notice 
jupyterlab-recents-2.0.0.tgz

Fetching URL: https://registry.yarnpkg.com/jupyterlab-recents
Fetching URL: https://registry.yarnpkg.com/jupyterlab-recents
Traceback (most recent call last):

  File "C:\ProgramData\Anaconda3\envs\topss\lib\site-packages\jupyterlab\debuglog.py", line 47, in debug_logging
    yield

  File "C:\ProgramData\Anaconda3\envs\topss\lib\site-packages\jupyterlab\labextensions.py", line 144, in start
    ans = self.run_task()

  File "C:\ProgramData\Anaconda3\envs\topss\lib\site-packages\jupyterlab\labextensions.py", line 191, in run_task
    [

  File "C:\ProgramData\Anaconda3\envs\topss\lib\site-packages\jupyterlab\labextensions.py", line 192, in <listcomp>
    install_extension(

  File "C:\ProgramData\Anaconda3\envs\topss\lib\site-packages\jupyterlab\commands.py", line 429, in install_extension
    return handler.install_extension(extension, pin=pin)

  File "C:\ProgramData\Anaconda3\envs\topss\lib\site-packages\jupyterlab\commands.py", line 654, in install_extension
    info = self._install_extension(extension, tempdir, pin=pin)

  File "C:\ProgramData\Anaconda3\envs\topss\lib\site-packages\jupyterlab\commands.py", line 1766, in _install_extension
    raise ValueError(msg)

ValueError: The extension "jupyterlab-recents" does not yet support the current version of JupyterLab.


Conflicting Dependencies:
JupyterLab              Extension      Package
>=3.4.7 <3.5.0          >=2.0.0 <3.0.0 @jupyterlab/application
>=5.4.7 <5.5.0          >=4.0.0 <5.0.0 @jupyterlab/coreutils
>=3.4.7 <3.5.0          >=2.0.0 <3.0.0 @jupyterlab/docmanager
>=3.4.7 <3.5.0          >=2.0.0 <3.0.0 @jupyterlab/mainmenu
>=6.4.7 <6.5.0          >=5.0.2 <6.0.0 @jupyterlab/services
>=3.4.7 <3.5.0          >=2.0.1 <3.0.0 @jupyterlab/statedb

Exiting application: lab

## Context
- Python package version:  3.8.13
- Jupyterlab-recents Extension version:

# Name                    Version                   Build  Channel
jupyterlab-recents        3.0.1              pyhd8ed1ab_0    conda-forge

```

```
C:\Users\PowerUser\Desktop>jupyter server extension list
Config dir: C:\Users\PowerUser\.jupyter

Config dir: C:\ProgramData\Anaconda3\envs\topss\etc\jupyter
    jupyterlab enabled
    - Validating jupyterlab...
      jupyterlab 3.4.7 ok
    nbclassic enabled
    - Validating nbclassic...
      nbclassic 0.4.3 ok
    notebook_shim enabled
    - Validating notebook_shim...
      notebook_shim  ok

Config dir: C:\ProgramData\jupyter
```

- Operating System and its version: Windows 10 
- Browser and its version: (not relevant.)

**Comment by fcollonval** [comment]
This project is a fork of the original extension. The consequence is that the NPM package (the one used when running the command `jupyter labextension install <package_name>`) has a different name: `@jlab-enhanced/recents`.

> The consequence is `jupyter labextension install jupyterlab-recents` is installing an older version.

That said, I highly discourage using `jupyter labextension install` for JupyterLab >=3. You should prefer the pip installation or conda installation.

**Comment by krassowski** [comment]
Closing as this extension was merged into JupyterLab 4.2 (see https://github.com/jupyterlab-contrib/jupyterlab-recents/issues/10 and https://github.com/jupyterlab/jupyterlab/pull/15483).
